src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.exceptions.InvalidFruitException;
+import core.basesyntax.service.exceptions.InvalidQuantityException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Storage {
+    private static final Map<Fruit, Integer> fruits = new HashMap<>();
+
+    public static Integer getQuantity(Fruit fruit) {
+        if (fruits.get(fruit) != null) {

```

Comment:
You are performing two hash table lookups here. First with fruits.get(fruit) != null and then return fruits.get(fruit). This is inefficient, consider storing the result in a local variable.

Reply:
I'm not sure how to solve it. What should I use instead of HashMap ?